src/lightning/pytorch/trainer/configuration_validator.py

```diff
@@ -16,7 +16,7 @@
 from lightning.fabric.utilities.warnings import PossibleUserWarning
 from lightning.pytorch.trainer.states import TrainerFn
 from lightning.pytorch.utilities.exceptions import MisconfigurationException
-from lightning.pytorch.utilities.imports import _LIGHTNING_GRAPHCORE_AVAILABLE
```

Comment:
This check shall live in this repository as it is the main topic...
If we have diplodocus in Lightning, that is fine...